### ISSUE TITLE
Issue with Tutorial_emulator

### ISSUE THREAD

**Issue by andreufont** [igmhub/ForestFlow#21]
Crashes in cell 7, probably easy to fix
`FileNotFoundError: [Errno 2] No such file or directory: '../data/emulator_models/mpg_hypercube.pt'`

**Comment by andreufont** [comment]
@jchavesmontero - probably needs another relative path?

**Comment by andreufont** [comment]
After fixing this by hand, the tutorial seems to work. I get a few warnings, not clear how much I should worry. For instance:

```
/nfs/pic.es/user/a/afontrib/Codes/ForestFlow/forestflow/P3D_cINN.py:609: UserWarning: Loading a pre-trained emulator
  warn("Loading a pre-trained emulator")
```

```
/nfs/pic.es/user/a/afontrib/Codes/ForestFlow/forestflow/P3D_cINN.py:472: UserWarning: Covariance matrix returned for p3d_arinyo.reshape(-1)
  warn("Covariance matrix returned for p3d_arinyo.reshape(-1)")
```

**Comment by andreufont** [comment]
by the way, given that the training of the emulator takes a while, I would hide it from the tutorial. We could leave it there (it is useful to see it), but I would protect it with something like:
```

if False:
    p3d_emu = P3DEmulator(...)
```

**Comment by jchavesmontero** [comment]
The issue with the path was addressed by [9e9d560](https://github.com/igmhub/ForestFlow/commit/9e9d5606101aa3557c288d8299223f89087e54f0). No worries about the warnings, both are fine. I am hiding the training now as well.
